### PR TITLE
Release Notes 8.0 iteration 1

### PR DESCRIPTION
**Pull request by hythloda**
First draft of release notes

### CODE REVIEW

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -117,24 +103,10 @@ testing {
 }
 ```
 
-#### Dependency verification metadata improvements
```
Comment (hythloda) [inline subprojects/docs/src/docs/release/notes.md:120]: @asodja can you add more context to this section, why is it helpful for a developer?

File: subprojects/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -117,24 +103,10 @@ testing {
...
+For more information about warning modes, see [JVM test suite](userguide/jvm_test_suite_plugin.html).
 
-The following nodes with dependency verification metadata file `verification-metadata.xml` now support a `reason` attribute:
 
-- the `trust` xml node under `trusted-artifacts`
-- the `md5`, `sha1`, `sha256` and `sha512` nodes under `component`
-
-#### Dependency verification CLI improvements
```
Comment (hythloda) [inline subprojects/docs/src/docs/release/notes.md:127]: @asodja can you add more context to this section, why is it helpful for a developer?

File: subprojects/docs/src/docs/release/notes.md [hunk 3]
```diff
@@ -203,6 +175,63 @@ plugins {
...
+#### Enhanced CodeNarc Plugin to automatically detects appropriate version for current Groovy runtime
+
+The [CodeNarc](https://codenarc.org/) project now publishes separate versions for use with Groovy 4.
+Gradle still currently ships with Groovy 3.
+
+To ensure future compatibility, the [CodeNarcPlugin](userguide/codenarc_plugin.html) now automatically detects the appropriate version of CodeNarc for the current Groovy runtime.
+
+You can still explicitly specify a CodeNarc version with the `toolVersion` property on the [CodeNarcExtension](dsl/org.gradle.api.plugins.quality.CodeNarcExtension.html#org.gradle.api.plugins.quality.CodeNarcExtension).
+
+#### Enhanced PMD and CodeNarc tasks to execute in parallel by default
```
Comment (hythloda) [inline subprojects/docs/src/docs/release/notes.md:224]: @tresat does this go in the Plugin section or JVM?

File: subprojects/docs/src/docs/release/notes.md [hunk 4]
```diff
@@ -117,24 +103,10 @@ testing {
...
-You can now use the `export-keys` flag to export all already trusted keys:
-
-```asciidoc
-./gradlew --export-keys
-```
-
-For more information, see [Exporting keys](userguide/dependency_verification.html#sec:local-keyring).
-
-### Kotlin DSL improvements
+###  Kotlin DSL
```
Comment (hythloda) [inline subprojects/docs/src/docs/release/notes.md:109]: @eskatos I think this section should be after the JVM? Willing to move it elsewhere, but that seems logical to me.

Comment (eskatos) [reply subprojects/docs/src/docs/release/notes.md:109]: Works for me :+1:

File: subprojects/docs/src/docs/release/notes.md (hunk 1 above)
Comment (asodja) [reply subprojects/docs/src/docs/release/notes.md:120]: Sure, will add.

File: subprojects/docs/src/docs/release/notes.md (hunk 2 above)
Comment (asodja) [reply subprojects/docs/src/docs/release/notes.md:127]: Acknowledged

File: subprojects/docs/src/docs/release/notes.md [hunk 5]
```diff
@@ -203,6 +175,63 @@ plugins {
...
+- the `md5`, `sha1`, `sha256` and `sha512` nodes under `component`
+
+#### Improved Dependency verification CLI
+
+You can now use the `export-keys` flag to export all already trusted keys:
+
+```asciidoc
+./gradlew --export-keys
+```
+
```
Comment (asodja) [inline subprojects/docs/src/docs/release/notes.md:206]:
```suggestion

This is useful when you want just export trusted keys without writing the verification metadata. 
```

File: subprojects/docs/src/docs/release/notes.md (hunk 1 above)
Comment (asodja) [reply subprojects/docs/src/docs/release/notes.md:120]: I added reviews suggestions down below.

File: subprojects/docs/src/docs/release/notes.md (hunk 2 above)
Comment (asodja) [reply subprojects/docs/src/docs/release/notes.md:127]: I added reviews suggestions down below. PTAL.

**Comment by hythloda** [comment]
@bot-gradle test and merge

**Pull request merged**